src/node_util.cc

```diff
@@ -174,6 +176,15 @@ void PromiseReject(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret.FromMaybe(false));
 }
 
+void SafeGetenvForJs(const FunctionCallbackInfo<Value>& args) {
+  CHECK(args[0]->IsString());
+  Local<String> envtag = args[0].As<String>();
+  String::Utf8Value strenvtag(envtag);
```

Comment:
Node has its own slightly more efficient Utf8Value implementation; just drop the `String::` part and pass in `args.GetIsolate()` as the first argument.  You can pass in `args[0]` as-is, no need to cast.

Reply:
done, thanks for the tip!